tests/config.py

```diff
@@ -41,7 +41,7 @@
 SCOPE_NAME = os.environ.get('SCOPE_NAME', 'test')
 
 
-TENANT_RESOURCES = {'limits.cpu': '2', 'limits.memory': '2Gi', 'requests.cpu': '1',
+TENANT_RESOURCES = {'limits.cpu': '4', 'limits.memory': '4Gi', 'requests.cpu': '1',
```

Comment:
This change don't have any effect. We deploy 2 endpoints with limits about 1Cpu and 1Gb, so limit for tenant was sufficient. Also our 2 nodes have only2 CPUs and some about 7GB RAM.